modules/userdirectory-canvas/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0"?>
```

Comment:
Not that this is completely accurate, but I don't think it's wrong on all of its findings:

```
% mvn dependency:analyze
…
[WARNING] Used undeclared dependencies found:
[WARNING]    org.apache.commons:commons-lang3:jar:3.8.1:compile
[WARNING]    commons-io:commons-io:jar:2.6:compile
[WARNING]    org.apache.commons:commons-collections4:jar:4.2:compile
[WARNING] Unused declared dependencies found:
[WARNING]    org.opencastproject:opencast-userdirectory:jar:7-SNAPSHOT:compile
[WARNING]    org.springframework.security:org.springframework.security.core:jar:3.1.4.RELEASE:compile
[WARNING]    org.springframework.security:org.springframework.security.web:jar:3.1.4.RELEASE:compile
[WARNING]    org.springframework.osgi:org.springframework.osgi.core:jar:1.2.1:compile
[WARNING]    org.springframework:org.springframework.jdbc:jar:3.1.4.RELEASE:compile
[WARNING]    org.springframework:org.springframework.beans:jar:3.1.4.RELEASE:compile
[WARNING]    org.springframework:org.springframework.core:jar:3.1.4.RELEASE:compile
[WARNING]    org.springframework:org.springframework.transaction:jar:3.1.4.RELEASE:compile
[WARNING]    org.easymock:easymock:jar:4.0.1:test
[WARNING]    org.slf4j:slf4j-log4j12:jar:1.7.21:test
[WARNING]    log4j:log4j:jar:1.2.17:test
```